modules/cactus-web/src/Dimmer/Dimmer.tsx

```diff
@@ -0,0 +1,66 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+export interface DimmerProps {
+  active: boolean
+  page?: boolean
+}
+
+type DimmerType = React.FC<DimmerProps> & {
+  DimmableContent: typeof DimmableContent
+}
+
+const DimmableContent: React.FC = ({ children }) => {
+  return <Container>{children}</Container>
+}
+
+const Dimmer: DimmerType = (props): React.ReactElement => {
+  const { active, page, children } = props
+  return (
+    <DimmerStyled active={active} page={page}>
+      {active && children}
+    </DimmerStyled>
+  )
```

Comment:
If you're going to use the `active` prop to conditionally render, I'd just go with conditionally rendering the whole thing:
```suggestion
  return active && (<DimmerStyled page={page}>{children}</DimmerStyled>)
```